scripts/sponsoring/1_add-modules-to-proxy.ts

```diff
@@ -0,0 +1,87 @@
+import { BigNumber } from '@ethersproject/bignumber';
+import { BytesLike } from '@ethersproject/bytes';
+import { time } from '@nomicfoundation/hardhat-network-helpers';
+import hre, { ethers } from 'hardhat';
+import CONFIG from '../../config/config.json';
+import {
+    IexecPoco1Delegate__factory,
+    IexecPoco2Delegate__factory,
+    IexecPocoAccessorsDelegate__factory,
+    TimelockController__factory,
+} from '../../typechain';
+import { Ownable__factory } from '../../typechain/factories/rlc-faucet-contract/contracts';
+import {
+    encodeModuleProxyUpdate,
+    logTxData,
+    printBlockTime,
+    printFunctions,
+} from '../upgrades/upgrade-helper';
+
+(async () => {
+    const chainId = (await ethers.provider.getNetwork()).chainId;
+    const deploymentOptions = CONFIG.chains[chainId].v5;
+    console.log('Link Boost functions to proxy:');
```

Comment:
```suggestion
    console.log('Link functions to proxy:');
```

Reply:
Thanks, see [Print proxy functions, read-only call with provider, fetch timelock delay & update logs/imports](https://github.com/iExecBlockchainComputing/PoCo/pull/91/commits/0103737fa2334da0140fe4a5a53de21162d5a66b) for all changes, lmk :)